### PR TITLE
Simpler way of implementing the progress bar

### PR DESCRIPTION
**Pull request by Robson**
Here's a different way of implementing the progress bar, which is a bit simpler.

### CODE REVIEW

**Comment by Robson** [comment]
I'm assuming it's fine to have HTML. If not, feel free to just use the characters instead:
▁
█

**Comment by Robson** [comment]
No activity after two weeks - closing.

File: index.js [hunk 1]
```diff
@@ -7,10 +7,9 @@ const progressBarOfThisYear = generateProgressBar()
 function generateProgressBar() {
     const progressBarCapacity = 30
     const passedProgressBarIndex = parseInt(progressOfThisYear * progressBarCapacity)
-    const progressBar = Array(progressBarCapacity)
-        .fill('▁')
-        .map((value, index) => index < passedProgressBarIndex ? '█' : value)
-        .join('')
+    const progressBar =
+        '&#9601;'.repeat(passedProgressBarIndex) +
```
Comment (liununu) [inline index.js:11]: Markdown supports HTML, but I think `▁` and `█` these symbols are more straightforward

File: index.js [hunk 2]
```diff
@@ -7,10 +7,9 @@ const progressBarOfThisYear = generateProgressBar()
 function generateProgressBar() {
     const progressBarCapacity = 30
     const passedProgressBarIndex = parseInt(progressOfThisYear * progressBarCapacity)
-    const progressBar = Array(progressBarCapacity)
-        .fill('▁')
-        .map((value, index) => index < passedProgressBarIndex ? '█' : value)
-        .join('')
+    const progressBar =
```
Comment (liununu) [inline index.js:10]: The passed progress bar should be `█`, instead of `▁` in the previous implementation

**Review by liununu — COMMENTED** [review]
Sorry for not seeing this PR, I have reviewed the commit.

**Comment by Robson** [comment]
No worries! I've fixed/changed those things and submitted a new pull request.

**Pull request closed without merging**